README.md

```diff
@@ -23,19 +23,46 @@ If the output is correct, the entire configuration is cleared.
 
 [ethereum.md](ethereum.md) is also capable of executing tests of arbitrary IELE transactions (subject to the fact that this first release of IELE is still
-built on top of an ethereum-like network layer). There are several differences in the syntax of the JSON test suite compared to the EVM test suite:
-
-* Users can specify the function called and the arguments to the function in blockchain tests. These fields are not currently part of the signed portion of the transaction,
-  for ease of compatibility with EVM tests. Transaction security will come in a future release.
-* Users can ask for gas checks in the test to be disabled by adding a `"checkGas": false` entry to the top-level JSON object.
-* VM tests call the function named `deposit`.
-* Users can specify a filename instead of a hexadecimal string when specifying the code of a contract, and the code will be pulled by assembling the file, which contains
-  IELE assembly. Filenames are relative to the directory containing the JSON file.
-* Blockchain tests have only one possible schedule, `Albe`.
-* Many fields that are checked by the Ethereum tests can be omitted or given arbitrary values, including:
-    * `blockHeader` does not check any values aside from those which are input to the VM, and the remaining can be given arbitrary values.
-    * `data` in a transaction can be an arbitrary value, but it still must match the signature of the transaction.
-    * `genesisBlockHeader` and `rlp` can be omitted.
+built on top of an ethereum-like network layer).
+
+The structure of the json files is as follows:
+
+* The top level is a json object, containing a list of key value pairs whose keys are the names of the tests to be executed and whose values are json objects containing an individual test case. The top-level fields in each test case are described below:
+  * "pre": a json object mapping account IDs (20-byte hexadecimal strings) to an object for each account representing the state of that account before the test starts.
+  * "blocks": a list of blocks the test should execute, each containing the folowing fields:
+    * "results": a list of results of each transaction in the block, each containing the following fields:
+      * "out": a list of integers representing the output of the top-level function call of the transaction.
+      * "status": the exit status of the top-level function call of the transaction.
+      * "gas": the amount of gas remaining after the transaction executes.
+      * "logs": the keccak256 hash of the rlp-encoded log entries generated by the transaction.
+      * "refund": the gas refund after the transaction executes.
+    * "transactions": a list of transactions to be executed, each containing the following fields:
+      * "nonce": the nonce of the sender of the transaction.
+      * "function": the name of the function to be called. Should be blank if the "to" field is blank.
```

Comment:
To be precise, instead of "blank" do you mean the value must be an empty string?